### PR TITLE
Add "validator type" to error (dgilperez/validates_zipcode#23)

### PR DESCRIPTION
**Pull request by nrser**
One line fiddle, seems to work, all tests pass (ruby 2.3.3).

### CODE REVIEW

**Comment by nrser** [comment]
Here's what I did for #23 if you want it.

File: lib/validates_zipcode/validator.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def validate_each(record, attribute, value)
       alpha2 = @country_code || record.send(@country_code_attribute)
 
       unless ValidatesZipcode::Zipcode.new(zipcode: value.to_s, country_alpha2: alpha2).valid?
-        record.errors.add(attribute, I18n.t('errors.messages.invalid_zipcode', value: value, default: 'Zipcode is invalid'))
+        record.errors.add(attribute, :invalid_zipcode, message: I18n.t('errors.messages.invalid_zipcode', value: value, default: 'Zipcode is invalid'))
```
Comment (dgilperez) [inline lib/validates_zipcode/validator.rb]: Hmm I'm curious: wouldn't `message` be unnecessary then? 

`record.errors.add(attribute, :invalid_zipcode, value: value, default: 'Zipcode is invalid'))`

Comment (dgilperez) [inline lib/validates_zipcode/validator.rb]: Not really, I think. It's better now, merging

**Comment by dgilperez** [comment]
Thanks! 🍻

**Comment by nrser** [comment]
no prob, thanks for the lib

**Pull request merged**